lighthouse-core/audits/largest-contentful-paint-element.js

```diff
@@ -65,7 +60,7 @@ class LargestContentfulPaintElement extends Audit {
 
     const details = Audit.makeTableDetails(headings, lcpElementDetails);
 
-    const displayValue = str_(UIStrings.displayValue, {itemCount: lcpElementDetails.length});
+    const displayValue = str_(i18n.UIStrings.displayValueElementsFound, {itemCount: lcpElementDetails.length});
```

Comment:
```suggestion
    const displayValue = str_(i18n.UIStrings.displayValueElementsFound, {nodeCount: lcpElementDetails.length});
```

should fix the tests